### PR TITLE
Added to VPC Gotchas

### PR DESCRIPTION
**Pull request by ljheidel**
Provided two gotchas about VPC VPNs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1131,11 +1131,14 @@ VPCs, Network Security, and Security Groups
 		-	e.g. A bug in the YAML parser used by the Ruby on Rails admin site is much less serious when the admin site is only visible to the private network and accessed through VPN.
 	-	Another common pattern (especially as deployments get larger, security or regulatory requirements get more stringent, or team sizes increase) is to provide a [bastion host](https://www.pandastrike.com/posts/20141113-bastion-hosts) behind a VPN through which all SSH connections need to transit.
 
+
```
Comment (ThanosBaskous) [inline README.md]: Nit: extra newline. Would you mind removing?

File: README.md [hunk 2]
```diff
@@ -1131,11 +1131,14 @@ VPCs, Network Security, and Security Groups
...
 -	❗Be careful when choosing your VPC IP CIDR block: If you are going to need to make use of [ClassicLink](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/vpc-classiclink.html), make sure that your private IP range [doesn’t overlap](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/vpc-classiclink.html#classiclink-limitations) with that of EC2 Classic.
 -	❗If you are going to peer VPCs, carefully consider the cost of of [data transfer between VPCs](https://aws.amazon.com/vpc/faqs/#Peering_Connections), since for some workloads and integrations, this can be prohibitively expensive.
+-	❗**Connecting VPCs Between Regions with VPNs**:  It's not possible to use the Amazon VPC VPN functionality to interconnect VPCs in multiple regions (for instance, to connect a VPC in us-east-1 to a VPC in us-west-2.)  To do this, you will have to use an instance running VPN software (such as Libreswan) or virtual VPN appliance in at least one  of the VPCs to create a tunnel between the VPCs.  You can, however, connect using a device or appliance in one region and Amazon's VPC VPN functionality in the other.
```
Comment (ThanosBaskous) [inline README.md]: Suggestion: "to use the Amazon VPC VPN functionality" -> "to use Amazon's Virtual Private Gateway functionality"

Comment (ThanosBaskous) [inline README.md]: Nit: one space between sentences.

Comment (ThanosBaskous) [inline README.md]: "virtual VPN appliance" - I think "software VPN appliance" might be better.

Comment (ThanosBaskous) [inline README.md]: "at least one  of the" - extra space

Comment (ThanosBaskous) [inline README.md]: Style: we use unicode double and single quotes throughout the guide - can you replace ' with ’?

File: README.md [hunk 3]
```diff
@@ -1131,11 +1131,14 @@ VPCs, Network Security, and Security Groups
...
+-	❗**Connecting Multiple VPCs to A Single IPSec VPN Endpoint**:  You cannot connect more than one VPC per region to a single IPSec endpoint (such as a router or a firewall on your network) using the VPC VPN functionality.  If you want to do this, you'll need to use an instance running VPN software (such as Libreswan) or a virtual VPN appliance in any VPC beyond your first VPC in that region.  You can use the VPC VPN functionality for your first VPC in the region, and then an instance or appliance for the second and subsequent VPCs, however this may increase complexity and you should consider using an instance or appliance for all VPCs.  (Note:  This applies across accounts.)
```
Comment (ThanosBaskous) [inline README.md]: nit: double spaces

**Comment by jlevy** [comment]
@richadams would you mind reviewing as well?

File: README.md (hunk 2 above)
Comment (ljheidel) [inline README.md]: Amazon seems to refer to the Virtual Private Gateway combined with the Customer Gateway simply as a VPN.  I understand your point though.  Maybe if I clarified it by saying initially that the VPN connection is formed by the combination of the Virtual Private Gateway and the Customer Gateway?

https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_VPN.html

**Comment by richadams** [comment]
👍  Content looks good, my only comment is the minor nit re: IPsec.

**Comment by jlevy** [comment]
@ljheidel would you be up for completing the rev on this so we can push it though? Thanks again!

**Comment by QuinnyPig** [comment]
@ljheidel If you can resolve the merges, I'm happy with it; closing this for now, please feel free to reopen once it merges cleanly.

**Pull request closed without merging**